tox.ini

```diff
@@ -13,37 +13,10 @@ minversion = 2.4
 
 [testenv]
 passenv = MAGIC_FOLDER_* PIP_* SUBUNITREPORTER_* USERPROFILE HOMEDRIVE HOMEPATH
-# Get "certifi" to avoid bug #2913. Basically if a `setup_requires=...` causes
-# a package to be installed (with setuptools) then it'll fail on certain
-# platforms (travis's OX-X 10.12, Slackware 14.2) because PyPI's TLS
-# requirements (TLS >= 1.2) are incompatible with the old TLS clients
-# available to those systems.  Installing it ahead of time (with pip) avoids
-# this problem.
```

Comment:
> Because it's no longer needed, I suppose?

This is definitely true. Looking at the [linked issue](https://tahoe-lafs.org/trac/tahoe-lafs/ticket/2913) it appears that it exists due to issues with travis, which I think has never been used for this project.

> It seems like the early certifi installation has been done away with? 

Yes and no. We don't explicitly install it, but since we do indirectly depend on it, it will be installed before we install our package (although not before we install things like `Twisted`, which it seems from the linked issue like this was about dealing with.

> This would be easier to understand as a separate change.

It is a prerequisite to this change. We are using tox to install our generated requirements files, which have hashes in them.
Since it uses a single `pip` invocation to install them, all the requirements need hashes.